pkg/storer/metrics.go

```diff
@@ -92,5 +92,5 @@ func (m getterWithMetrics) Get(ctx context.Context, address swarm.Address) (swar
 
 // captureDuration returns a function that returns the duration since the given start.
 func captureDuration(start time.Time) (elapsed func() float64) {
-	return func() float64 { return float64(time.Since(start).Nanoseconds()) }
+	return func() float64 { return time.Since(start).Seconds() }
```

Comment:
This should be also reflected in the Graphana.

Reply:
do we have the dashboards already?